src/net.cpp

```diff
@@ -1232,7 +1287,7 @@ void ThreadSocketHandler2(void* parg)
             //
             // Allow newbies to connect easily
             int64_t nTime = GetAdjustedTime();
-            if (nTime - pnode->nTimeConnected > 24)
+            if (nTime - pnode->nTimeConnected > 45)
```

Comment:
Just curious... what's the reason for this change?

Reply:
due to our older net code, bignodes sometimes do not send a version message within that period of connecting, so I upped it. #1548 optimizes this portion of net processing and actually gets rid of that setting altogether.